src/Scripting/CSharp/CSharpScriptCompiler.cs

```diff
@@ -67,13 +67,16 @@ public override Compilation CreateSubmission(Script script)
                     sourceReferenceResolver: script.Options.SourceResolver,
                     metadataReferenceResolver: script.Options.MetadataResolver,
                     assemblyIdentityComparer: DesktopAssemblyIdentityComparer.Default
-                ).WithTopLevelBinderFlags(BinderFlags.IgnoreCorLibraryDuplicatedTypes),
+                )),
                 previousSubmission,
                 script.ReturnType,
                 script.GlobalsType
             );
 
             return compilation;
         }
+
+        internal static CSharpCompilationOptions WithTopLevelBinderFlags(CSharpCompilationOptions options)
```

Comment:
`BinderFlags.IgnoreCorLibraryDuplicatedTypes` is internal hack to workaround issues with facades that duplicate types. This is used by both EE and Scripting. Scripting has IVT from compilers. Features/EditorFeatures can't use this flag directly since they do not have IVT from compilers (that would be even worse!). So this is the place where we can create the options and use them from EditorFeatures.